stable/kapacitor/templates/pvc.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.persistence.enabled }}
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+{{- if .Values.influxURL }}
```

Comment:
why the influxURL?

Reply:
following the same approach here: https://github.com/helm/charts/blob/master/stable/kapacitor/templates/deployment.yaml#L1

if you not set the influxURL the deployment will not be created. and if the deployment is not created let's apply the same for the PVC, to not have a not used resource hanging around